corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml

```diff
@@ -8,9 +8,10 @@
   since: "1.3"
   force: true
 
-- name: "Load multimedia files lazily"
-  description: "How should Commcare load the the multimedia files in the application. By default Commcare will try to load them instantly"
-  id: "lazy_load_multimedia_files"
+- name: "Load video files lazily"
+  description: "How should Commcare load the the video files in the application. By default Commcare will try to load them instantly"
+  id: "lazy_load_video_files"
+  toggle: OPTIONAL_MEDIA_IN_UPDATES
```

Comment:
I didn't know you could put toggle here? I don't see toggle elsewhere.

Reply:
Yeah, this works: https://github.com/dimagi/commcare-hq/blob/1c770f3c1e9487470349bda3ded952f5fdbcdbb4/corehq/apps/app_manager/views/apps.py#L185-L187

There are good [docs](urlhttps://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/commcaresettings-README.md) for commcare settings options.

Reply:
Yep. I luckily stumbled upon the docs in code 😅